charts/flyte/values.yaml

```diff
@@ -460,6 +461,15 @@ flyte:
 
     # -- Core propeller configuration
     core:
+      # -- follows the structure specified [here](https://pkg.go.dev/github.com/flyteorg/flytepropeller/manager/config#Config).
+      manager:
+        pod-application: "flytepropeller"
+        pod-template-container-name: "flytepropeller"
+        pod-template-name: "flytepropeller-template"
+        scan-interval: 10s
+        shard:
+          type: Hash
+          pod-count: 3
```

Comment:
No need to specify them here... in fact, this entire file will need a lot of cleanup since it's duplicating a bunch of things from flyte-core

Reply:
I kept the same as ^^^ where the pod-application, pod-template-container-name, and pod-template-name are still set here. I might be misunderstanding but the [deployments template generation](https://github.com/flyteorg/flyte/blob/0c4b9bd39a107a005227a763519ce5e49a97bfde/script/generate_helm.sh#L17) uses the flyte-core/values.yaml file but when generating for the [sandbox](https://github.com/flyteorg/flyte/blob/0c4b9bd39a107a005227a763519ce5e49a97bfde/script/generate_helm.sh#L14) we only use the flyte/values.yaml file. Maybe this should be cleaned up?

Reply:
My understanding is that helm will automatically pickup the `values.yaml` of the dependent chart + whatever overrides you have in your own `values.yaml`.. you can try removing them here and `make helm` to see if that generates the same...

Reply:
Absolutely, you are right. Seems to me like pretty helm master knowledge to me, my 30 minute tutorial didn't cover this. I removed it.